src/pages/dashboard/listings/[slug]/submissions.tsx

```diff
@@ -129,115 +186,174 @@ function BountySubmissions({ slug }: Props) {
+                            isDisabled={skip <= 0}
+                            leftIcon={<ChevronLeftIcon w={5} h={5} />}
+                            onClick={() =>
+                              skip >= length
+                                ? setSkip(skip - length)
+                                : setSkip(0)
+                            }
+                            size="sm"
+                            variant="outline"
+                          >
+                            Previous
+                          </Button>
+                          <Text color="brand.slate.400" fontSize="sm">
+                            <Text as="span" fontWeight={700}>
+                              {skip + 1}
+                            </Text>{' '}
+                            -{' '}
+                            <Text as="span" fontWeight={700}>
+                              {Math.min(skip + length, totalSubmissions)}
+                            </Text>{' '}
+                            of{' '}
+                            <Text as="span" fontWeight={700}>
+                              {totalSubmissions}
+                            </Text>{' '}
+                            Submissions
+                          </Text>
+                          <Button
+                            isDisabled={
+                              totalSubmissions <= skip + length ||
+                              (skip > 0 && skip % length !== 0)
+                            }
+                            onClick={() =>
+                              skip % length === 0 && setSkip(skip + length)
+                            }
+                            rightIcon={<ChevronRightIcon w={5} h={5} />}
+                            size="sm"
+                            variant="outline"
+                          >
+                            Next
+                          </Button>
+                        </>
+                      )}
+                    </Flex>
                   </>
                 )}
-              </Flex>
-            </>
-          )}
+              </TabPanel>
+              {bounty &&
+                bounty.id &&
+                bounty.isPublished &&
+                !bounty.isWinnersAnnounced && (
+                  <TabPanel px={0}>
+                    <ScountTable
+                      bountyId={bounty.id}
+                      scouts={scouts}
+                      setInvited={(userId: string) => {
+                        const scout = scouts.find(
+                          (scout) => scout.userId === userId,
+                        );
+                        if (!scout) return;
+                        if (scout) {
+                          scout.invited = true;
+                        }
+                        const scoutIndex = scouts.findIndex(
+                          (scout) => scout.userId === userId,
+                        );
+                        if (scoutIndex > -1 && scoutIndex < scouts.length) {
```

Comment:
so in this file, all numbers got subtracted by 1 for some reason. hence asking you: is the `-1` here intentional?

Reply:
all other numbers being subtracted one wasn't done by me, 
- but here findIndex returns -1 if element not found.